pkg/apps/application-list.jsx

```diff
@@ -180,7 +180,7 @@ export class ApplicationList extends React.Component {
             <Page>
                 <PageSection variant={PageSectionVariants.light}>
                     <Flex alignItems={{ default: 'alignItemsCenter' }}>
-                        <h2>{_("Applications")}</h2>
+                        <h2 className="pf-u-font-size-3xl">{_("Applications")}</h2>
```

Comment:
Changing the heading CSS dropped the size of this heading, so I imported the PF4 utility classes and use the appropriate font size adjustment as a class.